content-services/content-sources-backend
# Fixes 3215: Add module_hotfixes flag for repositories

## Summary

Add flag for disabling modularity filtering on repository basis.

## Testing steps

Create a repository with 'module_hotfix' set to true or false,  Fetching the repository should show it set to true or false depending on what you set it to.  Existing repos should be set to false, and new repos should default to false.

A snapshot's generated repo config should have it set to 0 or 1 depending on if the repo is set to false or true
